portal-web/test/functional/com/liferay/portalweb/paths/portlet/publications/PublicationsChanges.path

```diff
@@ -156,6 +156,34 @@
 	</td>
 <td></td>
 </tr>
+
+<!--SHARE ACCESS-->
+
+<tr>
+	<td>COPY_URL</td>
```

Comment:
Can you update the path title containing the `BUTTON`, such as COPY_URL_BUTTON to make it more clear?